substrate/frame/support/test/tests/pallet_ui/task_missing_condition.stderr

```diff
@@ -0,0 +1,5 @@
+error: missing `#[pallet::task_condition(..)]` attribute
+  --> tests/pallet_ui/task_missing_condition.rs:32:6
+   |
+32 |         fn foo() -> DispatchResult {
+   |            ^^^
```

Comment:
this one should already be covered by the pure parsing tests and can be omitted I think

Reply:
Keeping it for spans for now. Happy to remove once the discussion is final.